src/main/manager/mission-scripting.manager.ts

```diff
@@ -38,11 +38,14 @@ export class MissionScriptingManager {
     const scripts: { id: string; path: string }[] = []
 
     for (const asset of releaseAssets) {
-      if (asset.symlinkPath && asset.runOnStart) {
-        scripts.push({
-          id: asset.id,
-          path: asset.symlinkPath
-        })
+      for (const link of asset.links) {
+        // Ah should be runOnStart be a link property?
```

Comment:
☝️

Reply:
@JonathanTurnock I know very little about this feature.

Reply:
Yes this needs to be a property to indicate that the script should be run on mission start.

Like starting the DCS fiddle server see https://github.com/flying-dice/dcs-dropzone-registry/pull/34#discussion_r1912456479

That is then picked up by this manager and the dropzone MissionScripting file is rebuilt
https://github.com/flying-dice/dcs-dropzone-mod-manager/blob/main/src/main/manager/mission-scripting.manager.ts#L24